modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -58,7 +57,13 @@ public void createContainerWithShmVolume() {
             assertEquals("Shm mounts present", mounts.size(), 1);
 
             final InspectContainerResponse.Mount shmMount = mounts.get(0);
-            assertEquals("Shm mount source is correct", "/dev/shm", shmMount.getSource());
+
+            /* source path on Linux/OS X should be /dev/shm
+               source path on Windows is likely to be different: /host_mnt/c/dev/shm has been observed, but other paths
+               may be possible. As such, be liberal when asserting that the source path is correct.
+             */
+            assertTrue("Shm mount source is correct", shmMount.getSource().endsWith("/dev/shm"));
```

Comment:
This is not an especially clever assertion, but I hope is rigourous enough and commented to prevent this becoming a mystery in the future!